Lib/shutil.py

```diff
@@ -73,8 +73,14 @@ class RegistryError(Exception):
     and unpacking registries fails"""
 
 
-def copyfileobj(fsrc, fdst, length=16*1024):
-    """copy data from file-like object fsrc to file-like object fdst"""
+def copyfileobj(fsrc, fdst, length=None):
```

Comment:
```py
def copyfileobj(fsrc, fdst, length=16*1024):
```
I personally prefer this way in my code to concentrate 2 lines worth of code into one.

```py
    if not length:
        length = 16 * 1024
```
It looks a little nicer and no to mention saves lines of code that basically do the same thing.